envs/example/defaults-2.0.yml

```diff
@@ -228,6 +228,7 @@ neutron:
   network_vlan_ranges: ''
   tenant_network_type: vxlan
   arp_responder: false
+  l2_population: false
```

Comment:
```Allow overlapping IP support in Neutron. Attention: the following parameter MUST be set to False if Neutron is being used in conjunction with Nova security groups.```
According to openstack documentation, we should not turn this on if we are using nova secgroup. 
[root@controller-0 ~]# nova secgroup-list
+--------------------------------------+------------+------------------------+
| Id                                   | Name       | Description            |
+--------------------------------------+------------+------------------------+
| c23175ed-a68c-4759-a47d-88d580d0218e | default    | Default security group |
| 911aafd3-f3ba-4103-b383-166dd4c1d8dc | turtle-sec | turtle-sec             |
+--------------------------------------+------------+------------------------+